auth_backends/backends.py

```diff
@@ -234,8 +243,23 @@ def _to_language(locale):
 
 
 class EdXOAuth2(EdXBackendMixin, BaseOAuth2):
+    """
+    IMPORTANT: The oauth2 application must have access to the ``user_id`` scope in order
+    to use this backend.
+    """
+    # used by social-auth
+
     name = 'edx-oauth2'
 
+    DEFAULT_SCOPE = ['user_id', 'profile', 'email']
+    discard_missing_values = True
+    EXTRA_DATA = [('user_id', 'user_id', discard_missing_values)]
```

Comment:
How is `EXTRA_DATA` used?

Reply:
@douglashall: I added the comment above to address your question.  I also adjusted the commit comment, which you are welcome to review.